### PR TITLE
chore: Examples Updated to Remove Style Props

### PR DESCRIPTION
**Pull request by josh-bagwell**
## Summary

Fixes: #2725  

Removes style props from examples of components that have been updated in v11 to use new style utils. 

## Release Category
Documentation

---

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Where Should the Reviewer Start?

`/modules/preview-react/`
`/modules/react/`

### CODE REVIEW

File: modules/preview-react/radio/stories/examples/Basic.tsx [hunk 1]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import {FormField} from '@workday/canvas-kit-preview-react/form-field';
 import {RadioGroup} from '@workday/canvas-kit-preview-react/radio';
 import {Flex} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const formfieldInputStyles = createStyles({
+  width: '200px',
+});
```
Comment (RayRedGoose) [inline modules/preview-react/radio/stories/examples/Basic.tsx:9]:
```suggestion
import {createStyles, px2rem} from '@workday/canvas-kit-styling';

const formfieldInputStyles = createStyles({
  width: px2rem(200),
});
```

File: modules/preview-react/radio/stories/examples/Custom.tsx [hunk 2]
```diff
@@ -2,6 +2,17 @@ import React from 'react';
 import {FormField} from '@workday/canvas-kit-preview-react/form-field';
 import {RadioGroup} from '@workday/canvas-kit-preview-react/radio';
 import {Flex} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base} from '@workday/canvas-tokens-web';
+
+const styleOverrides = {
+  formfieldInputStyles: createStyles({
+    width: '200px',
+  }),
```
Comment (RayRedGoose) [inline modules/preview-react/radio/stories/examples/Custom.tsx:11]:
```suggestion
import {createStyles, px2rem} from '@workday/canvas-kit-styling';
import {base} from '@workday/canvas-tokens-web';

const styleOverrides = {
  formfieldInputStyles: createStyles({
    width: px2rem(200),
  }),
```

File: modules/preview-react/radio/stories/examples/Inverse.tsx [hunk 3]
```diff
@@ -4,13 +4,20 @@ import {RadioGroup} from '@workday/canvas-kit-preview-react/radio';
...
+import {base, system} from '@workday/canvas-tokens-web';
 
 const StyledFormField = styled(FormField)<StyledType>({
   legend: {
     color: colors.frenchVanilla100,
   },
 });
 
+const parentContainerStyles = createStyles({
+  backgroundColor: base.blueberry400,
```
Comment (RayRedGoose) [inline modules/preview-react/radio/stories/examples/Inverse.tsx:17]: should this be system token? I guess we don't need to support of using base tokens if there is system alternative existing

File: modules/react/button/stories/button/examples/ExternalHyperlinkInverse.tsx [hunk 4]
```diff
@@ -2,9 +2,16 @@ import React from 'react';
 
 import {ExternalHyperlink} from '@workday/canvas-kit-react/button';
 import {Box} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const parentContainerStyles = createStyles({
+  backgroundColor: base.blueberry400,
```
Comment (RayRedGoose) [inline modules/react/button/stories/button/examples/ExternalHyperlinkInverse.tsx:9]:
```suggestion
  backgroundColor: system.color.bg.primary.default,
```

File: modules/react/button/stories/button/examples/HyperlinkInverse.tsx [hunk 5]
```diff
@@ -2,9 +2,16 @@ import React from 'react';
 
 import {Hyperlink} from '@workday/canvas-kit-react/button';
 import {Box} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const parentContainerStyles = createStyles({
+  backgroundColor: base.blueberry400,
```
Comment (RayRedGoose) [inline modules/react/button/stories/button/examples/HyperlinkInverse.tsx:9]:
```suggestion
  backgroundColor: system.color.bg.primary.default,
```

File: modules/react/button/stories/button/examples/PrimaryInverse.tsx [hunk 6]
```diff
@@ -7,9 +7,17 @@ import {
   relatedActionsVerticalIcon,
   caretDownIcon,
 } from '@workday/canvas-system-icons-web';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const parentContainerStyles = createStyles({
+  gap: system.space.x4,
+  backgroundColor: base.blueberry400,
```
Comment (RayRedGoose) [inline modules/react/button/stories/button/examples/PrimaryInverse.tsx:15]: same here

File: modules/react/checkbox/stories/examples/Indeterminate.tsx [hunk 7]
```diff
@@ -1,6 +1,13 @@
 import React from 'react';
 import {Checkbox} from '@workday/canvas-kit-react/checkbox';
 import {Box} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
+
+const styleOverrides = createStyles({
+  marginLeft: system.space.x8,
```
Comment (RayRedGoose) [inline modules/react/checkbox/stories/examples/Indeterminate.tsx:8]:
```suggestion
  marginInlineStart: system.space.x8,
```

File: modules/react/combobox/stories/examples/Autocomplete.tsx [hunk 8]
```diff
@@ -44,6 +45,20 @@ const AutoCompleteInput = createSubcomponent(TextInput)({
   return <Combobox.Input as={Element} {...elemProps} />;
 });
 
+const styleOverrides = {
+  inputGroupInner: createStyles({
+    width: 20,
```
Comment (RayRedGoose) [inline modules/react/combobox/stories/examples/Autocomplete.tsx:50]:
```suggestion
    width: px2rem(20),
```

File: modules/react/combobox/stories/examples/Autocomplete.tsx [hunk 9]
```diff
@@ -17,6 +17,7 @@ import {FormField} from '@workday/canvas-kit-preview-react/form-field';
 import {StyledMenuItem} from '@workday/canvas-kit-react/menu';
 import {LoadingDots} from '@workday/canvas-kit-react/loading-dots';
 import {InputGroup, TextInput} from '@workday/canvas-kit-react/text-input';
+import {createStyles} from '@workday/canvas-kit-styling';
```
Comment (RayRedGoose) [inline modules/react/combobox/stories/examples/Autocomplete.tsx:20]:
```suggestion
import {createStyles, px2rem} from '@workday/canvas-kit-styling';
```

File: modules/react/combobox/stories/examples/Autocomplete.tsx [hunk 10]
```diff
@@ -44,6 +45,20 @@ const AutoCompleteInput = createSubcomponent(TextInput)({
...
+  inputGroupInner: createStyles({
+    width: 20,
+    transition: 'opacity 100ms ease',
+  }),
+  loadingDots: createStyles({
+    display: 'flex',
+    transform: 'scale(0.3)',
+  }),
+  comboboxMenuList: createStyles({
+    maxHeight: 200,
```
Comment (RayRedGoose) [inline modules/react/combobox/stories/examples/Autocomplete.tsx:58]:
```suggestion
    maxHeight: px2rem(200),
```

File: modules/react/combobox/stories/examples/Autocomplete.tsx [hunk 11]
```diff
@@ -95,12 +110,12 @@ export const Autocomplete = () => {
         <InputGroup>
           <InputGroup.Input as={FormField.Input.as(AutoCompleteInput)} />
           <InputGroup.InnerEnd
+            cs={styleOverrides.inputGroupInner}
             pointerEvents="none"
-            style={{opacity: loader.isLoading ? 1 : 0, transition: 'opacity 100ms ease'}}
-            width={20}
+            style={{opacity: loader.isLoading ? 1 : 0}}
```
Comment (RayRedGoose) [inline modules/react/combobox/stories/examples/Autocomplete.tsx:115]: should this line use system opacity tokens?

File: modules/react/icon/stories/examples/AccentIconList.tsx [hunk 12]
```diff
@@ -3,11 +3,31 @@ import * as CanvasAccenttIcons from '@workday/canvas-accent-icons-web';
...
+    flexDirection: 'column',
+    alignItems: 'center',
+    gap: system.space.x6,
+  }),
+  firstChildContainer: createStyles({
+    flexWrap: 'wrap',
+  }),
+  secondChildContainer: createStyles({
+    alignItems: 'center',
+    width: `max(320px,20%)`,
```
Comment (RayRedGoose) [inline modules/react/icon/stories/examples/AccentIconList.tsx:24]:
```suggestion
    width: `max(px2rem(320),20%)`,
```

File: modules/react/icon/stories/examples/AppletIconList.tsx [hunk 13]
```diff
@@ -3,11 +3,31 @@ import * as CanvasAppletIcons from '@workday/canvas-applet-icons-web';
...
+    flexDirection: 'column',
+    alignItems: 'center',
+    gap: system.space.x6,
+  }),
+  firstChildContainer: createStyles({
+    flexWrap: 'wrap',
+  }),
+  secondChildContainer: createStyles({
+    alignItems: 'center',
+    width: `max(320px,20%)`,
```
Comment (RayRedGoose) [inline modules/react/icon/stories/examples/AppletIconList.tsx:24]: `px2rem`

File: modules/preview-react/radio/stories/examples/Inverse.tsx [hunk 14]
```diff
@@ -4,13 +4,20 @@ import {RadioGroup} from '@workday/canvas-kit-preview-react/radio';
...
 import {styled, StyledType} from '@workday/canvas-kit-react/common';
 import {colors} from '@workday/canvas-kit-react/tokens';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
 
 const StyledFormField = styled(FormField)<StyledType>({
   legend: {
     color: colors.frenchVanilla100,
   },
 });
```
Comment (mannycarrera4) [inline modules/preview-react/radio/stories/examples/Inverse.tsx:14]: can you update this to remove the styled component and just use create styles

File: modules/react/select/stories/examples/FetchingDynamicItems.tsx [hunk 15]
```diff
@@ -4,6 +4,12 @@ import {Select, useSelectModel} from '@workday/canvas-kit-react/select';
 import {Flex} from '@workday/canvas-kit-react/layout';
 import {PrimaryButton} from '@workday/canvas-kit-react/button';
 import {useMount} from '@workday/canvas-kit-react/common';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const parentContainerStyles = createStyles({
+  flexDirection: 'column',
+  maxWidth: 300,
+});
```
Comment (RayRedGoose) [inline modules/react/select/stories/examples/FetchingDynamicItems.tsx:12]:
```suggestion
import {createStyles, px2rem} from '@workday/canvas-kit-styling';

const parentContainerStyles = createStyles({
  flexDirection: 'column',
  maxWidth: px2rem(300),
});
```

File: modules/react/select/stories/examples/MenuHeight.tsx [hunk 16]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import {FormField} from '@workday/canvas-kit-preview-react/form-field';
 import {Select} from '@workday/canvas-kit-react/select';
 import {Box} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const selectCardStyles = createStyles({
+  maxHeight: 200,
+});
```
Comment (RayRedGoose) [inline modules/react/select/stories/examples/MenuHeight.tsx:9]:
```suggestion
import {createStyles, px2rem} from '@workday/canvas-kit-styling';

const selectCardStyles = createStyles({
  maxHeight: px2rem(200),
});
```

File: modules/react/table/stories/examples/Basic.tsx [hunk 17]
```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 
 import {Table} from '@workday/canvas-kit-react/table';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base} from '@workday/canvas-tokens-web';
+
+const tableHeaderStyles = createStyles({
+  backgroundColor: base.soap100,
+});
```
Comment (RayRedGoose) [inline modules/react/table/stories/examples/Basic.tsx:9]:
```suggestion
import {createStyles} from '@workday/canvas-kit-styling';
import {system} from '@workday/canvas-tokens-web';

const tableHeaderStyles = createStyles({
  backgroundColor: system.color.bg.alt.softer,
});
```

File: modules/react/table/stories/examples/BasicWithHeading.tsx [hunk 18]
```diff
@@ -3,22 +3,33 @@ import React from 'react';
...
 import {useUniqueId} from '@workday/canvas-kit-react/common';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const styleOverrides = {
+  parentContainerStyles: createStyles({
+    marginBottom: system.space.x4,
+  }),
+  tableHeaderStyles: createStyles({
+    backgroundColor: base.soap100,
```
Comment (RayRedGoose) [inline modules/react/table/stories/examples/BasicWithHeading.tsx:14]:
```suggestion
    backgroundColor: system.color.bg.alt.softer,
```

File: modules/react/checkbox/stories/examples/Inverse.tsx [hunk 19]
```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import {Checkbox} from '@workday/canvas-kit-react/checkbox';
 import {Flex} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+const styleOverrides = createStyles({
+  gap: system.space.x4,
+  backgroundColor: base.blueberry400,
```
Comment (mannycarrera4) [inline modules/react/checkbox/stories/examples/Inverse.tsx:9]: for things that use blueberry400, do we want to change this to be `brand.primary`?

File: modules/react/icon/stories/examples/AccentIconList.tsx [hunk 20]
```diff
@@ -3,11 +3,31 @@ import * as CanvasAccenttIcons from '@workday/canvas-accent-icons-web';
...
+const styleOverrides = {
+  parentContainer: createStyles({
+    flexDirection: 'column',
+    alignItems: 'center',
+    gap: system.space.x6,
+  }),
+  firstChildContainer: createStyles({
+    flexWrap: 'wrap',
+  }),
+  secondChildContainer: createStyles({
```
Comment (mannycarrera4) [inline modules/react/icon/stories/examples/AccentIconList.tsx:22]: is there something more semantic we can use other than `first, second` for the class names?

File: modules/react/icon/stories/examples/AppletIconList.tsx [hunk 21]
```diff
@@ -3,11 +3,31 @@ import * as CanvasAppletIcons from '@workday/canvas-applet-icons-web';
...
+    flexDirection: 'row',
+    gap: system.space.x3,
+    padding: system.space.x3,
+  }),
+};
```
Comment (mannycarrera4) [inline modules/react/icon/stories/examples/AppletIconList.tsx:29]: same here, what is it trying to style?

File: modules/react/select/stories/examples/Alert.tsx [hunk 22]
```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import {FormField} from '@workday/canvas-kit-preview-react/form-field';
 import {Select} from '@workday/canvas-kit-react/select';
 import {Flex} from '@workday/canvas-kit-react/layout';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const parentContainerStyles = createStyles({
```
Comment (mannycarrera4) [inline modules/react/select/stories/examples/Alert.tsx:7]: what makes sense, `parentContainer` or `flexStyles` ?

File: modules/react/checkbox/stories/examples/Inverse.tsx (hunk 19 above)
Comment (mannycarrera4) [reply modules/react/checkbox/stories/examples/Inverse.tsx:9]: reference Raisa's comment above

Comment (RayRedGoose) [reply modules/react/checkbox/stories/examples/Inverse.tsx:9]: yeah, there is more other instances below than I commented

File: modules/preview-react/radio/stories/examples/Inverse.tsx (hunk 14 above)
Comment (josh-bagwell) [reply modules/preview-react/radio/stories/examples/Inverse.tsx:14]: Done

File: modules/preview-react/radio/stories/examples/Basic.tsx (hunk 1 above)
Comment (josh-bagwell) [reply modules/preview-react/radio/stories/examples/Basic.tsx:9]: Thanks for catching this Raisa!

File: modules/preview-react/radio/stories/examples/Inverse.tsx (hunk 3 above)
Comment (josh-bagwell) [reply modules/preview-react/radio/stories/examples/Inverse.tsx:17]: Yup!

File: modules/react/checkbox/stories/examples/Indeterminate.tsx (hunk 7 above)
Comment (josh-bagwell) [reply modules/react/checkbox/stories/examples/Indeterminate.tsx:8]: Great catch!

File: modules/react/combobox/stories/examples/Autocomplete.tsx (hunk 11 above)
Comment (josh-bagwell) [reply modules/react/combobox/stories/examples/Autocomplete.tsx:115]: Yeah I went ahead and updated

File: modules/react/icon/stories/examples/AccentIconList.tsx (hunk 12 above)
Comment (josh-bagwell) [reply modules/react/icon/stories/examples/AccentIconList.tsx:24]: Thank you Raisa!

File: modules/react/checkbox/stories/examples/Inverse.tsx (hunk 19 above)
Comment (josh-bagwell) [reply modules/react/checkbox/stories/examples/Inverse.tsx:9]: I would say default would be using `system.color.primary.default` then when it's branded, we would use `brand`

File: modules/react/icon/stories/examples/AccentIconList.tsx (hunk 20 above)
Comment (josh-bagwell) [reply modules/react/icon/stories/examples/AccentIconList.tsx:22]: I went ahead and updated this

File: modules/react/icon/stories/examples/AppletIconList.tsx (hunk 21 above)
Comment (josh-bagwell) [reply modules/react/icon/stories/examples/AppletIconList.tsx:29]: Updated this naming

File: modules/react/select/stories/examples/Alert.tsx (hunk 22 above)
Comment (josh-bagwell) [reply modules/react/select/stories/examples/Alert.tsx:7]: Either or. 

I went with `parentContainerStyles` since it's applied on the parent container

**Review by mannycarrera4 — APPROVED** [review]

File: modules/react/select/stories/examples/WithIcons.tsx [hunk 23]
```diff
@@ -8,6 +8,16 @@ import {
...
 } from '@workday/canvas-system-icons-web';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const styleOverrides = {
+  formfieldInputStyles: createStyles({
+    width: '300px',
+  }),
+  selectCardStyles: createStyles({
+    maxHeight: '200px',
+  }),
```
Comment (RayRedGoose) [inline modules/react/select/stories/examples/WithIcons.tsx:19]:
```suggestion
import {createStyles, px2rem} from '@workday/canvas-kit-styling';

const styleOverrides = {
  formfieldInputStyles: createStyles({
    width: px2rem(300),
  }),
  selectCardStyles: createStyles({
    maxHeight: px2rem(200),
  }),
```

**Pull request merged**